cluster-provision/gocli/cmd/run.go

```diff
@@ -114,6 +115,7 @@ func NewRunCommand() *cobra.Command {
 	run.Flags().Bool("enable-psa", false, "Pod Security Admission")
 	run.Flags().Bool("single-stack", false, "enable single stack IPv6")
 	run.Flags().Bool("enable-audit", false, "enable k8s audit for all metadata events")
+	run.Flags().StringArrayVar(&usbDisks, "usb", []string{}, "size of the emulate USB disk to pass to the node")
```

Comment:
My question is more about the end result for testing usb devices (emulated) in kubevirt.

So, the default is empty. Will we have an image that we can use in CI with something like `export KUBEVIRT_PROVIDER_EXTRA_ARGS="--usb 20M --usb 30M"`  ?

Reply:
@victortoso if we need we could set the flag and the environmental variables when creating the kubevirtci instance on the CI.

Reply:
Yes, we can do that in automation